Tasks/AzurePowerShellV3/TryMakingModuleAvailable.ps1

```diff
@@ -0,0 +1,33 @@
+[CmdletBinding()]
+param (
```

Comment:
how this works in case of "latest" version selection ?

Reply:
In case of latest targetVersion would be "" so we make a check and return. In the existing logic it iterates over the folder and add the latest one in the module path.